corehq/apps/reminders/urls.py

```diff
@@ -3,38 +3,14 @@
 from django.conf.urls import url
 
 from corehq.apps.reminders.views import (
-    CreateScheduledReminderView,
-    EditScheduledReminderView,
-    RemindersListView,
-    CreateComplexScheduledReminderView,
     KeywordsListView,
     AddStructuredKeywordView,
     EditStructuredKeywordView,
     AddNormalKeywordView,
     EditNormalKeywordView,
-    BroadcastListView,
-    CreateBroadcastView,
-    EditBroadcastView,
-    CopyBroadcastView,
-    ScheduledRemindersCalendarView,
-    rule_progress,
 )
 
 urlpatterns = [
-    url(r'^list/$', RemindersListView.as_view(), name=RemindersListView.urlname),
```

Comment:
Does it seem likely that someone has bookmarked or otherwise may access one of these URLs after they're removed? Would it make sense to redirect to the appropriate/equivalent page for the new framework rather than removing these entirely?

Reply:
it seems unlikely to me, but these urls have also been returning 404 for a while now for many projects (as soon as the migration completed for a project, these urls would 404)